ac/ac-induction/src/TestPart/Test.jsx

```diff
@@ -45,9 +46,14 @@ export default ({
       feedback ? 'listIndexTestWithFeedback' : 'listIndexTest'
     );
     if (feedback) {
+      logger({
+        type: 'TestFB:' + groupingValue,
```

Comment:
The grouping value is already part of the log automatically. It does not make sense to include it in the type. We want to use type in the logs for possible actions: "looking at an example", "submitting an answer", "going to the next phase".